pkg/version/version.go

```diff
@@ -0,0 +1,28 @@
+package version
```

Comment:
we dont need to add this, this is part of `flytestdlib`
https://github.com/lyft/flytestdlib/blob/master/version/version.go
you already have a dependency on flytestdlib

Reply:
Again I do not see the change? We should remove this file right? and just use the one in flytestdlib